### PR TITLE
Sigv4 Blog Post PR

### PR DESCRIPTION
**Pull request by erichsueh3**
This PR adds a blog post for the `sigv4` extension, and discusses why it was added, how to use it, and how it affects Collector configuration.

### CODE REVIEW

**Review by mhausenblas — COMMENTED** [review]
Overall LGTM, two thoughts:

1.  You promise "why it was added" though that's not explained
2. The post leads with the SigV4 extension and mentions as an aside the change in the PRW (is it worth a separate post? should that combo be the center of attention?)

**Comment by erichsueh3** [comment]
> Overall LGTM, two thoughts:
> 
> 1. You promise "why it was added" though that's not explained
> 2. The post leads with the SigV4 extension and mentions as an aside the change in the PRW (is it worth a separate post? should that combo be the center of attention?)

For the first point, I will reword it to be more clear with what the rest of the post explains, something like "what it is used for" instead. As for the second point, I don't think it is worth a separate post, but I will change the title of the post since the migration to using sigv4 + PRW is a large focus.

**Review by mhausenblas — APPROVED** [review]
